src/internal/connector/data_collections.go

```diff
@@ -89,18 +92,23 @@ func verifyBackupInputs(sel selectors.Selector, mapOfUsers map[string]string) er
 	return nil
 }
 
-// createCollections - utility function that retrieves M365
+// ---------------------------------------------------------------------------
+// Exchange
+// ---------------------------------------------------------------------------
+
+// createExchangeCollections - utility function that retrieves M365
 // IDs through Microsoft Graph API. The selectors.ExchangeScope
-// determines the type of collections that are stored.
-// to the GraphConnector struct.
-func (gc *GraphConnector) createCollections(
+// determines the type of collections that are retrieved.
+func (gc *GraphConnector) createExchangeCollections(
 	ctx context.Context,
 	scope selectors.ExchangeScope,
 ) ([]*exchange.Collection, error) {
-	var errs *multierror.Error
+	var (
+		errs           *multierror.Error
```

Comment:
For standardization, var errs should be changed to `error` in line with the other vars in the file.

Reply:
Err handling in this file is all over the place.  I'll fix it up how you've proposed in a follow-up PR, to isolate the changes from this one.